adresse/controllers/service.classic.php

```diff
@@ -165,4 +169,95 @@ function update(){
     $rep->data = array('success' => ''.$result, 'type'=>$typeRes, 'message' => $message);
     return $rep;
   }
+
+  function export(){
+
+    $test='#^[0-9]{1}[0-9AB]{1}[0-9]{3}$#';
+    $filterParams = array();
+    $project = $this->param('project');
+    $repository = $this->param('repository');
+    $insee = $this->param('insee');
+    $option = $this->param('opt');
+
+    if(!$project){
+      $rep = $this->getResponse('json');
```

Comment:
```suggestion

    $rep = $this->getResponse('json');
    if(!$project){
```
Tu peux définir ta réponse une fois en dehors des tests